connector_details.go

```diff
@@ -45,6 +48,7 @@ type ConnectorDetailsResponse struct {
 		} `json:"status"`
 		Config ConnectorConfigResponse `json:"config"`
 	} `json:"data"`
+	// SourceSyncDetails not implemented yet. https://fivetran.height.app/T-114130
```

Comment:
Just wondering ... as https://fivetran.height.app is only accessible by internal fivetran audience, is it useful to mention it in the open source project code?

Reply:
Good catch, I'll fix it. Thanks @fivetran-joliveira. I'll keep only the ticket number for our internal reference.